docker/worker/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM opensuse:13.2
 MAINTAINER Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>
 
-RUN useradd -m geekotest && \
-    zypper --non-interactive --gpg-auto-import-keys in ca-certificates-mozilla curl && \
-    zypper ar -f http://download.opensuse.org/repositories/devel:openQA:stable/openSUSE_13.2/devel:openQA:stable.repo && \
```

Comment:
So what it will be? Are you ok with using devel now? If so, I'm happy to merge it as is.